code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -81,10 +81,10 @@
 			else
 				to_chat(H, span("danger", "You sense great disturbance to your physical body!"))
 		else
-			user.visible_message("<span class='danger'>[user] attacked [src] with [I] waking [t_him] up!</span>", \
+			visible_message("<span class='danger'>[user] attacked [src] with [I] waking [t_him] up!</span>", \
```

Comment:
Why did you remove the user from here? The second line is written with the user as the receiver, so this makes no sense now.